components/container/index.jsx

```diff
@@ -77,8 +77,7 @@ function maybeRenderWarning(locationIsInUsersPod, noControlError, podRootIri) {
 export default function Container({ iri }) {
   const { sessionRequestInProgress } = useSession();
   const [resourceUrls, setResourceUrls] = useState(null);
-  const { data: authenticatedProfile, error: authenticatedProfileError } =
-    useAuthenticatedProfile();
+  const authenticatedProfile = useAuthenticatedProfile();
```

Comment:
is there something wrong with using the authenticatedProfileError? is that why its getting removed or is not getting the right error types?

Reply:
The reason is that `useFullProfile` doesn't return an error and just leaves the profile as null if something fails, otherwise it will return whatever values it can fetch in the profile object. I could change this to return an error if necessary.